docs/_topic_map.yml

```diff
@@ -47,12 +47,14 @@ Topics:
     - Name: Upgrading from Older Versions
       File: upgrading
 ---
-Name: Getting Started
+Name: Getting Started with Helm
```

Comment:
Question: Is the product name actually `Helm`. I would use whatever `{product-title}` is set to. Maybe it can be reused here as well?

Reply:
It's been referred to as Helm elsewhere, but excellent point to use {product-title} as has been used elsewhere.

Reply:
@j-white Could you verify I should use {product-title} in the _topic_map.yml file? Other instances of "Helm" in the file did not use {product-title}. Tx.

Reply:
@mvrueden It looks like I can't use a variable for the product name in the _topic_map.yml file.